package.json

```diff
@@ -89,7 +89,7 @@
 		"eslint-plugin-import": "^2.27.5",
 		"eslint-plugin-prettier": "^4.2.1",
 		"jest": "^25.5.4",
-		"nodemon": "^1.19.4",
+		"nodemon": "^2.0.22",
```

Comment:
why is this changed?

Reply:
![Screenshot 2023-05-23 at 4 04 00 PM](https://github.com/icgc-argo/platform-api/assets/79996555/59518648-42d8-4e96-9bb6-2d4f3d638b58)

It's from Dan's ticket when he ran a npm audit fix two weeks ago.

Reply:
@demariadaniel Dan, can you take a look at the change to nodemon version update, before we merge this to main? Thank you

Reply:
Yes this is patching a security vulnerability